src/puppetlabs/puppetdb/command.clj

```diff
@@ -233,9 +233,9 @@
 
 (defmacro upon-error-throw-fatality
   [& body]
-  `(try+
+  `(try
     ~@body
-    (catch Throwable e#
+    (catch Exception e#
```

Comment:
If we want to catch :pre and post here, I think we might need to catch AssertionError too (it's a Throwable...).